packages/razzle-plugin-scss/README.md

```diff
@@ -103,6 +109,31 @@ See [node-sass options](https://github.com/sass/node-sass#options) to override c
 
 ---
 
+### sassModules: _object_
+
+default
+
+```js
+{
+  dev: {
+    sourceMap: true,
+    includePaths: [paths.appNodeModules],
+    modules: true
+  },
+  prod: {
+    sourceMap: false,
```

Comment:
This needs to be set to `true`, otherwise production build fails when referencing imports using `resolve-url-loader`.